src/components/strategies/create/useCreateStrategy.ts

```diff
@@ -46,7 +47,20 @@ export const useCreateStrategy = () => {
   const order1 = useOrder(templateStrategy?.order1);
   const order0 = useOrder(templateStrategy?.order0);
   const [isProcessing, setIsProcessing] = useState<boolean>(false);
+  const { marketPricePercentage: buyMarketPricePercentage } =
+    useMarketIndication({
```

Comment:
it seems to me we are using this hook in to many places. Can you give some context as to why we need this in different places?

Reply:
of course, we need the market indication for both create strategy & edit strategy.
LimitRangeSection component is common for both of them so I'm using over there.
Also I needed market price indication for gtm events only for create strategy event, that is why I also added the custom hook also to useCreateStrategy. let me know if that make sense to you.